person.rb

```diff
@@ -1,8 +1,21 @@
-class Person
+# Nameable class
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+# Person class
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
-  def initialize(age, parent_permission: true, name: 'Unknown')
+  def correct_name
+    @name
+  end
```

Comment:
- [OPTIONAL] The `correct_name` method should come after the initialize method, please kindly fix that.

Reply:
Ok, Noted with thanks. I am correcting it immediately @Mihndim2020